src/Menu.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+/*
+ jiaoyan testpage
+ */
+var MENU = [
+    {did: '1', rid: '1', price: '$49.99',  dname: 'Pad Thai', discription: 'Chicken, and shrimp; stir-fried with rice noodles, egg, scallions, bean sprouts, and ground peanuts.'},
+    {did: '2', rid: '1', price: '$49.99',  dname: 'crab rangoon', discription: 'Chicken, and shrimp; stir-fried with rice noodles, egg, scallions, bean sprouts, and ground peanuts.'},
+    {did: '3', rid: '1', price: '$49.99',  dname: 'downpling', discription: 'Chicken, and shrimp; stir-fried with rice noodles, egg, scallions, bean sprouts, and ground peanuts.'},
+    {did: '4', rid: '1', price: '$49.99',  dname: 'noodle', discription: 'Chicken, and shrimp; stir-fried with rice noodles, egg, scallions, bean sprouts, and ground peanuts.'},
+    {did: '5', rid: '1', price: '$49.99',  dname: 'fried rice', discription: 'Chicken, and shrimp; stir-fried with rice noodles, egg, scallions, bean sprouts, and ground peanuts.'},
+    {did: '6', rid: '1', price: '$49.99',  dname: 'chicken wings', discription: 'Chicken, and shrimp; stir-fried with rice noodles, egg, scallions, bean sprouts, and ground peanuts.'}
+];
+/*this is the parent of menu page*/
+class Menu extends React.Component {
+    render() {
+        return (
+            <div>
+              <h1>The Menu</h1>
+              <MenuTable menu = {MENU}/>
+                /*Todo: Add button switch to MyOrder Page*/
+            </div>
+        )
+    }
+}
+
+/*get each row shown in menu table*/
+class MenuTable extends React.Component {
```

Comment:
Can you change this to functional Component? I did not see any use case in the future that it will manipulate state. Did this also for CuisionRow, OrderTable & OrderRow.